exercises/03_reverse/reverse.go

```diff
@@ -1,4 +1,18 @@
 package main
 
+import "fmt"
+
+func Reverse(n int) (r int) {
```

Comment:
The function should have been called `reverse` (all lower-case). The function name called in the test file was correct.